<launch>
  <!-- Start the simulation with the specified world -->
  <include file="$(find tiago_iaslab_simulation)/launch/start_simulation.launch">
    <arg name="world_name" value="iaslab_assignment2" />
  </include>

  <!-- Navigation stack -->
  <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch"/>

  <!-- Run the get_straightline_node -->
  <node pkg="tiago_iaslab_simulation" type="get_straightline_node" name="get_straightline_node" output="screen"/>

  <include file="$(find tiago_iaslab_simulation)/launch/apriltag2.launch"/>

      <!-- Start tables_node -->
  <node pkg="assignment2_package" type="tables_node" name="tables_node" output="screen"/>

    <!-- Start node_B -->
  <node pkg="assignment2_package" type="node_B" name="node_B" output="screen"/>

    <!-- Start node_C -->
  <node pkg="assignment2_package" type="node_C" name="node_C" output="screen"/>

    <!-- Start node_A -->
  <node pkg="assignment2_package" type="node_A" name="node_A" output="screen"/>

</launch>